tests/data/parser.json

```diff
@@ -1,1963 +1,2424 @@
 [
   {
-    "test": "0001",
+    "test": "0001", 
```

Comment:
Original formatting is quite odd, seems like file was written manually. However, I've pushed custom JSON formatter which tries to be close to the original formatting as much as possible.